### PR TITLE
Simplify price in CompTaskInfo

### PR DESCRIPTION
**Pull request by Krigpl**
`price` setter and getter ain't no used

### CODE REVIEW

File: golem/task/taskkeeper.py [hunk 1]
```diff
@@ -55,32 +55,21 @@ class WrongOwnerException(Exception):
 class CompTaskInfo:
     def __init__(self, header: TaskHeader, price: int):
         self.header = header
-        self._price, self.subtask_price = 0, 0  # lints and typing
-        self.price = price
-        self.requests = 1
-        self.subtasks = {}
-        # TODO Add concent communication timeout. Issue #2406
```
Comment (maaktweluit) [inline golem/task/taskkeeper.py:62]: Is this also no longer needed? Is the issue closed?

**Review by maaktweluit — CHANGES_REQUESTED** [review]
LGTM!

One small q, approve on reply!

File: golem/task/taskkeeper.py (hunk 1 above)
Comment (Krigpl) [reply golem/task/taskkeeper.py:62]: This isn't relevant to this PR and I didn't touch that, the issue is not closed (as you can check by yourself) and I dunno if it's still needed. I might have missed the point of your question :P

Comment (maaktweluit) [reply golem/task/taskkeeper.py:62]: my question was: "Why is this removed"

Now i see its moved.. /shy

No questions remain ;)

**Review by maaktweluit — APPROVED** [review]

**Pull request merged**